shannon/testnet-beta/README.md

```diff
@@ -17,6 +18,10 @@ track of things we'll need in the genesis files along with supporting details.
 - **Validator4**: `pokt1a5f5hxgaxg83y6s5w8k47dpwylkms4jjjrl9l6`
 - **Validator5**: `pokt19csy3xzhpnzk8eu9ykjjzxk3vzma8gxz5rx3dy`
 
+## Seeds
+
+Currently supported list of seed nodes can be found in the [seeds](./seeds) file.
```

Comment:
Is there anything to now regarding how to use this file?

Reply:
No, this is just a text file that contains a list of "official" seed nodes. Just one node for now, hehe.